src/config/external-applications.md

```diff
@@ -0,0 +1,51 @@
+# External Applications
+
+## Programming Languages
+
+> The Void repositories have a number of python and lua packages. If possible,
+> try to install packages from the Void repos using xbps-install or use xbps-src
+> to package your python or lua application. Packaging your application with
+> xbps-src will keep your system clean. If you send your package upstream it
+> will allow other Void users to benifit and allow them to help you maintain
+> your package. To learn how to package for xbps-src see the
+> [manual](https://github.com/void-linux/void-packages/blob/master/Manual.md)
+> and [contributing
+> instructions](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md).
+
+To keep your system lean, Void splits packages and their devel packages. If you
+install a program via the languages package manager (pip, gem, etc.) or compile
+via source you may need to install the programming language's devel package to
+ensure your program builds.
+
+| Language | Package Manager                | Void Package  |
+|----------|--------------------------------|---------------|
+| Python   | pip, anaconda, virtualenv, etc | python3-devel |
+| Ruby     | gem                            | ruby-devel    |
+| lua      | luarocks                       | lua-devel     |
+
+## Restricted Packages
+
+Some packages have legal restrictions on their distribution (ie. discord), may
+be too large, or have another condition that makes it difficult for the void
+team to distribute. These packages are maintained but not distributed. As such,
+they must be built locally. For more information see the page on [restricted
+packages](../repositories/restricted.md).
+
+## Non-x86_64 Arch
+
+The entire Void build system is using the x86_64 architecture. xbps-src allows
+us to cross-compile for other architectures and their glibc or musl variant.
+However, some packages (ie. libreoffice) do not support cross-compilation. For
+these appplications, you must build the application locally on the machine you
+want to use the package. For compiling and installing xbps-src applications see
+the [building instructions](../packages/building.md).
+
+## Flatpak
+
+Flatpak is another method for installing external proprietary applications on
+linux. For information on using flatpak with Void Linux, see the [official
+flatpak documentation](https://flatpak.org/setup/Void%20Linux/).
```

Comment:
You need to capitalize Linux and Flatpak in this paragraph. And I think "external" in this case doesn't need to be mentioned. If it is proprietary, it is external already.

Perhaps add "proprietary applications, such as Spotify and Steam."

@bobertlo do you think we have to mention that Steam also has a package? With zero guarantees, as it stands.